api/userList/index.ts

```diff
@@ -16,11 +16,11 @@ async function handleUserListRoute(req, res) {
         res.send(response.data)
     } catch (error) {
         const errReport = JSON.stringify({
```

Comment:
Do we still need JSON.stringify()? The responses are converted into json by the bodyParser.json() middleware anyway (or should be)

Reply:
There seems to be inconsistencies across endpoints regarding response stringification. I will remove it here and see if it works.